src/chain/error.rs

```diff
@@ -53,6 +55,12 @@ pub enum ChainError {
     #[error("Tendermint error")]
     Tendermint(#[from] TendermintError),
 
+    #[error("Tendermint_rpc error")]
+    TendermintRpc(#[from] tendermint_rpc::Error),
```

Comment:
We can rely on the existing `TendermintError` if we switch the tendermint-rpc to the same version that cosmrs is using right now.